### ISSUE TITLE
[steamshovel] add new view options (side view, top view) (Trac #1166)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1160]
it is often useful to have the camera aligned with the xyz axes. while it is easy to produce such an alignment with the scripting engine (e.g. window.gl.cameraLoc = (1000, 0, 0)), it is seems also useful to add correspondings options to the view menu.

Migrated from https://code.icecube.wisc.edu/ticket/1166
```json
{
    "status": "closed", 
    "changetime": "2015-10-08T17:35:56", 
    "description": "it is often useful to have the camera aligned with the xyz axes. while it is easy to produce such an alignment with the scripting engine (e.g. window.gl.cameraLoc = (1000, 0, 0)), it is seems also useful to add correspondings options to the view menu.", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1444325756692207", 
    "component": "combo core", 
    "summary": "[steamshovel] add new view options (side view, top view)", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-18T19:57:52", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151005T15:43:33: hdembinski changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151008T17:35:56: 
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 138393:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138393"
> added simple camera view options, closes #1160
> ```


* hdembinski changed status from "accepted" to "closed"
